interface/batchcom/batchcom.php

```diff
@@ -1,303 +1,281 @@
+                <label for="age_upto text-center"><?php echo xlt('to'); ?></label>
+                <input name="age_upto" size="2" type="num" class="form-control" placeholder="<?php echo xla("any"); ?>">
+            </div>
+            <div class="col-md-3 well form-group">
+                <label for="app_s"><?php echo xlt('Appointment within') ?>:</label>
+                    <input type="text" class="datepicker form-control" name="app_s" placeholder="<?php echo xla('any date'); ?>">
+                    <div class="text-center"><?php echo xlt('to'); ?></div>
+                    <input type="text" class="datepicker form-control" name="app_e" placeholder="<?php echo xla('any date'); ?>">
+            </div>
+            <!-- later gator    <br>Insurance: <SELECT multiple NAME="insurance" Rows="10" cols="20"></SELECT> -->
+            <div class="col-md-3 well form-group">
+                <label for="app_s"><?php echo xlt('Seen within')?>:</label>
+                    <input type="text" class="datepicker form-control" name="seen_since" placeholder="<?php echo xla('any date'); ?>">
+                    <div class="text-center"><?php echo xlt('to'); ?></div>
+                    <input type="text" class="datepicker form-control" name="seen_before" placeholder="<?php echo xla('any date'); ?>">
+            </div>
+        </div>
+        <div class="email row form-group">
+            <div class="col-md-6 col-md-offset-3 well">
+                <div class="col-md-6">
+                    <label for="email_sender"><?php echo xlt('Email Sender'); ?>:</label>
+                    <input class="form-control" type="text" name="email_sender" placeholder="your@email.email">
+                </div>
+                
+                <div class="col-md-6">
+                    <label for="email_subject"><?php echo xlt('Email Subject'); ?>:</label>
+                    <input class="form-control" type="text" name="email_subject" placeholder="From your clinic">
+                </div>
+                <div class="col-md-12">
+                    <label for="email_subject"><?php echo xlt('Email Text, Usable Tag: ***NAME*** , i.e. Dear ***NAME***{{Do Not translate the ***NAME*** elements of this constant.}}'); ?>:</label>
+                </div>
+                <div class="col-md-12">
+                    <textarea class="form-control" name="email_body" id="" cols="40" rows="8"></textarea>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-4 col-md-offset-4 text-center form-group">
+                <input type="hidden" name="form_action" value="process">
+                <input type="submit" name="submit" class="btn btn-primary form-control" value="<?php echo xla("Process (can take some time)"); ?>">
+            </div>
+        </div>
+    </form>
+</main>
+</body>
+
+<script>
+    (function() {
```

Comment:
qucik question @TheToolbox ,
Is this the same as using the jquery $(document).ready(function() {
?